docs/src/references/c8y-configuration-management.md

```diff
@@ -144,12 +151,15 @@ The `c8y_configuration_plugin` reports progress and errors on its `stderr`.
 When a configuration file is successfully downloaded from the cloud,
 the `c8y_configuration_plugin` service notifies this update over MQTT.
 
-* Each message provides the path to the freshly updated file as in `{ "changedFile": "/etc/tedge/tedge.toml" }`.
-* The messages are published on `tedge/configuration_change`.
+* The notification messages are published on the topic `tedge/configuration_change/{type}`,
+  where `{type}` is the type of the configuration file that have been updated,
+  for instance `tedge/configuration_change/tedge.toml`
+* Each message provides the path to the freshly updated file as in `{ "path": "/etc/tedge/tedge.toml" }`.
 
-## Internals
-
-Points that still need to be addressed:
-
-* Ability to give a type name to a configuration file.
-* What if the target file doesn't exist? Is this is an error? If not what user, group and mod are to be used?
\ No newline at end of file
+Note that:
+* If no specific type has been assigned to a configuration file, then the path to this file is used as its type.
+  Update notifications for that file are then published on the topic `tedge/configuration_change/{path}`,
+  for instance `tedge/configuration_change//etc/tedge/mosquitto-conf/c8y-bridge.conf`.
```

Comment:
That double `//` after `tedge/configuration_change` is how it should be? Shouldn't we remove that redundant `/`?

Reply:
This is the topic actually used when the path is used as type.

Reply:
If the `path` is set as `/etc/tedge/mosquitto-conf/c8y-bridge.conf` and no `type` is given, `/etc/tedge/mosquitto-conf/c8y-bridge.conf` will be treated as `type`. Therefore, the notification topic has double slashes. If the user doesn't want it, they should define the type.